build/hygiene.js

```diff
@@ -241,6 +250,18 @@ function hygiene(some, linting = true, secrets = true) {
 				}
 			})))
 		);
+		// --- Start Positron ---
+		streams.push(
+			eslintMochaOnlyHook(((message, isError) => {
+				if (isError) {
+					console.error(message);
+					errorCount++;
+				} else {
+					console.warn(message);
+				}
+			}))
+		);
+		// --- End Positron ---
```

Comment:
If the eslint mocha hook is causing issues, this can be commented out as a temporary measure.